runtime/src/main/java/org/corfudb/runtime/object/transactions/AbstractTransactionalContext.java

```diff
@@ -79,6 +79,7 @@
     public final TransactionBuilder builder;
 
     /**
+     * TODO remove this!
```

Comment:
Lots of reasons to have it at this level, for example, metering won't give you per transaction level information, which you may very well want to introspect on (like self aborting if the transaction is too old...)